### PR TITLE
nvme-cli: update to 2.0, adopt

### PR DESCRIPTION
**Pull request by subnut**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by subnut** [comment]
Since @phf seems to be inactive, I thought I'd adopt this package.

@phf, if you are reading this, is it okay if I adopt this package?

File: srcpkgs/nvme-cli/template [hunk 1]
```diff
@@ -1,28 +1,19 @@
...
+revision=1
+build_style=meson
+configure_args="-Ddocs=man"
+hostmakedepends="pkg-config"
+makedepends="libnvme-devel zlib-devel json-c-devel libuuid-devel libhugetlbfs-devel"
 short_desc="NVMe management command line interface"
-maintainer="Peter Froehlich <peter.hans.froehlich@gmail.com>"
-license="GPL-2.0-only"
+maintainer="Subhaditya Nath <sn03.general@gmail.com>"
+license="LGPL-2.0-or-later"
```
Comment (leahneukirchen) [inline srcpkgs/nvme-cli/template:11]: This doesn't match the LICENSE in the repo.

Comment (subnut) [reply srcpkgs/nvme-cli/template:11]: https://github.com/linux-nvme/nvme-cli/blob/5a36baba322e7979175ed9dfeede0a96e29efc37/meson.build#L5

Comment (leahneukirchen) [reply srcpkgs/nvme-cli/template:11]: https://github.com/linux-nvme/nvme-cli/blob/5a36baba322e7979175ed9dfeede0a96e29efc37/LICENSE

Comment (leahneukirchen) [reply srcpkgs/nvme-cli/template:11]: See https://github.com/linux-nvme/nvme-cli/issues/1508, revert this line,

**Comment by leahneukirchen** [comment]
`/usr/etc/udev/rules.d` is wrong.

**Comment by subnut** [comment]
@leahneukirchen Fixed.

**Comment by leahneukirchen** [comment]
Please remove 70-nvmf-autoconnect.rules it tries to call systemd.

**Pull request merged**